### PR TITLE
Fix CVE-2021-45985 on memcached and ntopng

### PR DESCRIPTION
**Pull request by 0xba1a**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

Patch lua source that comes along with `memcached` and `ntopng` packages

###### Change Log  

- CVE-2021-45985

###### Does this affect the toolchain?  

**NO**

###### Links to CVEs  
- https://web.nvd.nist.gov/view/vuln/detail?vulnId=CVE-2021-45985

###### Test Methodology

- local build

### CODE REVIEW

File: SPECS/ntopng/ntopng.spec [hunk 1]
```diff
@@ -12,6 +12,7 @@ URL:            https://www.ntop.org/
 Source0:        %{name}-%{version}.tar.gz
 #Source1:       https://github.com/ntop/nDPI/archive/%{nDPIver}.tar.gz
 Source1:        nDPI-%{nDPIver}.tar.gz
+Patch1:			CVE-2021-45985.patch
```
Comment (oliviacrain) [inline SPECS/ntopng/ntopng.spec:15]: Nit: inconsistent spacing with other lines

File: SPECS/memcached/memcached.spec [hunk 2]
```diff
@@ -7,14 +7,15 @@
...
-Release:        2%{?dist}
+Release:        3%{?dist}
 License:        BSD
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 URL:            https://www.memcached.org/
 Source0:        https://www.memcached.org/files/%{name}-%{version}.tar.gz
 Source1:        memcached.sysconfig
 Patch0:         memcached-unit.patch
+Patch1:			CVE-2021-45985.patch
```
Comment (oliviacrain) [inline SPECS/memcached/memcached.spec:18]: Nit: Inconsistent spacing with other lines

**Review by oliviacrain — APPROVED** [review]
Looks good. Can we file a bug in ADO to remind us to investigate the removal of these vendored lua versions?

**Comment by 0xba1a** [comment]
@oliviacrain Raised Bug: https://microsoft.visualstudio.com/OS/_workitems/edit/44241419

**Pull request merged**